test/ttmlir/Dialect/TTNN/multiple_add_with_loc.mlir

```diff
@@ -3,7 +3,7 @@
 #loc = loc("test_ops.py:17_0_0":0:0)
```

Comment:
I added this test, make optimizer enabled here.

Reply:
Done.